hub/powertoys/command-palette/publish-extension.md

```diff
@@ -1,72 +1,585 @@
+```
+
+You can run this command from the "Run commands" command in Command Palette, from the command line, or from the Run dialog.
+
+## Guide to Microsoft Store publishing
+
+Publishing to the Microsoft Store provides your extension with wide reach across Windows devices and automatic update delivery to users. This guide walks you through the complete process from setting up your Partner Center account to building MSIX packages and submitting your extension for certification. You'll learn how to prepare your extension's manifest files, create the required bundle packages, and navigate the Partner Center submission workflow to get your extension published successfully.
+
+> [!NOTE]
+> This guide provides basic Microsoft Store publishing steps specific to Command Palette extensions. For comprehensive Microsoft Store publishing guidance, including detailed submission requirements, certification processes, and best practices, see [Publish Windows apps and games](https://learn.microsoft.com/windows/apps/publish/).
+
+### Prerequisites
+
+> [!IMPORTANT]
+> **What is Partner Center?**
+> Partner Center is Microsoft's portal for app developers to manage Microsoft Store submissions, track analytics, and handle app certification.
+
+- [Register as a Windows app developer in Partner Center](https://learn.microsoft.com/en-us/windows/apps/publish/partner-center/partner-center-developer-account)
+- Create all required app icons and ensure they're properly sized ([Create icons using Visual Studio's asset generation tool](https://learn.microsoft.com/en-us/windows/apps/design/style/iconography/visual-studio-asset-generation))
+
+### Set up Microsoft Store
+
+1. Navigate to the [Microsoft Partner Center](https://partner.microsoft.com/dashboard/home).
+1. Under **Workspaces**, select **Apps and games**.
+1. Select **+ New Product**.
+1. Select **MSIX or PWA app**.
+1. Create a name or reserve a product name.
+1. Start the submission and complete as much as you can until you reach the **Packages** section.
+1. In the left navigation, under **Product Management**, select **Product identity**.
+1. Note the `Package/Identity/Name`, `Package/Identity/Publisher`, and `Package/Properties/PublisherDisplayName` for the following steps.
+
+### Prepare the extension
+
+1. In your IDE, open `<ExtensionName>\Package.appxmanifest`.
+1. Update the publisher and publisher display name:
+
+```xml
+<Identity
+    Name="TemplateCmdPalExtension" <!-- Replace with Package/Identity/Name -->
+    Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" <!-- Replace with Package/Identity/Publisher -->
+    Version="0.0.1.0" />
+
+  <Properties>
+    <DisplayName>TemplateDisplayName</DisplayName> <!-- Replace with the reserved name -->
+    <PublisherDisplayName>A Lone Developer</PublisherDisplayName> <!-- Replace with Package/Properties/PublisherDisplayName -->
+    <Logo>Assets\StoreLogo.png</Logo> <!-- Update -->
+  </Properties>
+```
```

Comment:
Think this could use some extra clarity. The steps say to just update publisher and publisher display name, but the comments imply that all of these values need to be updated.

Also for StoreLogo, might be good to say that if logos were generated via the generation tool, then this value should already be correct.

This may be overly cautious, but people tend to get bogged down with partner center identity stuff, so might be good to over-explain.